### ISSUE TITLE
Add type instability info

### ISSUE THREAD

**Issue by davidanthoff** [julia-vscode/ChromeProfileFormat.jl#8]
I just tried to use the new profiler support in the VS Code extension for real, and I'm realizing we are missing a major piece that is present in things like ProfileViewer.jl: some visual indication in the flame graph where there is a type instability. That is actually really one of the most useful pieces in ProfileViewer...

So I guess the first question is whether there is anything in the `.cpuprofile` format that would allow us to include that kind of info, and then whether it would show up say via a different color in the VS Code visualizer.

@vchuravy, any ideas?

**Comment by vchuravy** [comment]
One could highlight call that go through `jl_apply_generic` or some such. But type-instability is not really a notion that the profiler should be aware of. To actually analyse the behaviour you would need something like Cthulhu

**Comment by davidanthoff** [comment]
But somehow that is done in ProfileView.jl, I’d just like to see the same thing :)

**Comment by pfitzseb** [comment]
Yup, that's basically how ProfileView (actually, FlameGraphs) does it:
```
function status(sf::StackFrame)
    st = UInt8(0)
    if sf.from_c && (sf.func === :jl_invoke || sf.func === :jl_apply_generic)
        st |= runtime_dispatch
    end
    if sf.from_c && startswith(String(sf.func), "jl_gc_")
        st |= gc_event
    end
    if !sf.from_c && sf.func === :eval_user_input && endswith(String(sf.file), "REPL.jl")
        st |= repl
    end
    return st
end
```
https://github.com/timholy/FlameGraphs.jl/blob/47b62b67701dd7d0e1978204f65289ee906b8652/src/graph.jl#L138-L150

**Comment by davidanthoff** [comment]
So the question is whether the `.cpuprofile` format has some field that we could use to carry that information, and then whether the flame graph implementation would show different colors there...
